strawberry_django_plus/type.py

```diff
@@ -545,3 +545,10 @@ def wrapper(cls):
         )
 
     return wrapper
+
+
+@dataclasses.dataclass
+class FullClean:
+    exclude: Optional[list[str]] = None
+    validate_unique: bool = True
+    validate_constraints: bool= True
```

Comment:
I was going to comment that this should probably be defined inside the `mutation/resolvers.py` directly, but I suggested using a typeddict instead =P. That typeddict can be defined there, probably with a name of `FullCleanOptions`

Reply:
thanks for reviewing. will fix and submit again.